play-json/shared/src/main/scala/Json.scala

```diff
@@ -18,15 +16,15 @@ object Json {
    * @param input a String to parse
    * @return the JsValue representing the string
    */
-  def parse(input: String): JsValue = JacksonJson.parseJsValue(input)
+  def parse(input: String): JsValue = StaticBinding.parseJsValue(input)
```

Comment:
Static binding, using either the JS or JVM underlying functions, according the env-specific artifact